docs/EN_US/ECLWatch/ECLWa_mods/SessFaq.xml

```diff
@@ -0,0 +1,269 @@
+    ECLWatch in incognito mode, or delete the browser cookie cache, in order
+    to log in as a different user?</para>
+
+    <para><emphasis role="strong">A</emphasis>: No, with our new custom HPCC
+    Login screen, you can select previously used IDs or enter a different one.
+    You can have as many user sessions active at any time as permitted by your
+    system’s resources. </para>
+     </listitem>
+
+	      <listitem>
+    <para><emphasis role="strong">Q</emphasis>: Is there an option to stay
+    logged in forever?</para>
+
+    <para><emphasis role="strong">A</emphasis>: No. </para>
+     </listitem>
+
+	      <listitem>
+    <para><emphasis role="strong">Q</emphasis>: Is there any option to not
+    time out from inactivity?</para>
+
+    <para><emphasis role="strong">A</emphasis>: No.</para>
+     </listitem>
+
+	      <listitem>
+    <para><emphasis role="strong">Q</emphasis>: Will I lose data if I get
+    automatically logged out?</para>
+
+    <para><emphasis role="strong">A</emphasis>: No. You will not get logged
+    out, your session will get locked. Anything typed into any fields (such as
+    a search box) that has not been submit or entered could potentially be
+    lost. However, since the session is only locked, it is unlikely that any
+    data will be lost. The ECL IDE automatically saves your work so nothing
+    should get lost. </para>
+     </listitem>
+
+      <listitem>
+    <para><emphasis role="strong">Q</emphasis>: Will my queued and scheduled
+    workunits run when I am locked out?</para>
+
+    <para><emphasis role="strong">A</emphasis>: Yes, the session only applies
+    to ESP/ECL IDE and ESP/ECLWatch communications.</para>
+     </listitem>
+
+	      <listitem>
+    <para><emphasis role="strong">Q</emphasis>: Will HPCC command line
+    utilities such as testsocket or ecl be affected?</para>
+
+    <para><emphasis role="strong">A</emphasis>: No. To the users they will
+    function exactly as before session management.</para>
+     </listitem>
+
+	
+	      <listitem>
+    <para><emphasis role="strong">Q</emphasis>: Does auto refresh in ECLWatch
+    reset the session expiration timer?</para>
+
+    <para><emphasis role="strong">A</emphasis>: No. Only user initiated
+    actions extend the timeout.</para>
+     </listitem>
+
+	
+	      <listitem>
+    <para><emphasis role="strong">Q</emphasis>: If I am logged in to the same
+    account using multiple tabs in a browser, or multiple instances of the
+    same browser, can I get logged out of one but not the others?</para>
+
+    <para><emphasis role="strong">A</emphasis>: No, user activity is tracked
+    via credentials. Activity in one tab or instance will extend the session
+    for all.</para>
+     </listitem>
+
+	
+	      <listitem>
+    <para><emphasis role="strong">Q</emphasis>: If I am logged in to the same
+    account using different browsers (ie Firefox and Chrome), do they share
```

Comment:
i.e., means "that is" and  e.g., mean "for example."  Both need punctuation when you use them.
This sentence should say:  (e.g., Firefox and Chrome)